cfgov/v1/models/browse_filterable_page.py

```diff
@@ -68,7 +72,19 @@ class NewsroomLandingPage(BrowseFilterablePage):
 
     objects = PageManager()
 
-    def base_query(self, hostname):
-        """ Modify the base query for Newsroom to exclude research reports """
-        reports = [x[0] for x in dict(ref.categories)['Research Report']]
-        return AbstractFilterPage.objects.live_shared(hostname).exclude(categories__name__in=reports)
+    @classmethod
+    def eligible_categories(cls):
+        categories = dict(ref.categories)
+        return sorted(itertools.chain(*(
+            dict(categories[category]).keys()
+            for category in ('Blog', 'Newsroom')
+        )))
```

Comment:
@richaagarwal Thanks for reviewing asking this, I figured you'd have a better sense of what this logic is supposed to do.

The logic in this PR tries to say: "`NewsroomLandingPage` pages only ever show posts that are tagged with the categories listed under "Blog" and "Newsroom" in the lists at [`v1.util.ref.categories`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/util/ref.py#L51)."

The logic you linked to is slightly different (I think) -- it says: "When someone submits a form, where the choice of topics comes from the list at [`v1.util.ref.page_types`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/util/ref.py#L27), we have to convert the topics from that list to their subcategories in `v1.util.ref.categories`."

For me at least there's some confusion about the difference between "topics" and "categories" and "page types". Why are there separate lists for `page_types` and `categories` at all? Could [the set of category choices in `FilterableListForm`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/forms.py#L118) just use the set of `categories`, not `page_types`?